src/scene/sprite/Sprite.ts

```diff
@@ -172,101 +157,11 @@ export class Sprite extends Container implements View
      * The local bounds of the sprite.
      * @type {rendering.Bounds}
      */
-    get bounds()
```

Comment:
These changes break rendering of sprites with trimmed textures, I assume.

Reply:
this should be all good! i merged it so that bounds always returns the trimmed bounds if they exist.
This keeps things the same as they were. But will have a think about how to distinguish between the two. Something like `bounds` for when users measure and perhaps `viewBounds` which factors in trimmed textures which we can use internally (eg for filters measuring)

Reply:
The sprite should be rendered based on the `texture.trim` bounds (which doesn't appear to be the case in this PR), but the `bounds` of the sprite itself should not use `texture.trim` so that is has the same bounds like the same texture would have if it was untrimmed (this is correct in this PR). If we had a visual test for a trimmed texture, I'd expect it to fail and render stretched with this PR.

Reply:
let me put up a visual test for this in the PR

Reply:
test added - and trim code is back in. Thanks for clarifying!